packages/flutter_chat/example/lib/main.dart

```diff
@@ -19,18 +19,19 @@ class MyApp extends StatelessWidget {
         useMaterial3: true,
       ),
       home: const MyHomePage(),
-      // FutureBuilder(
-      // future: Firebase.initializeApp(
-      // options: DefaultFirebaseOptions.currentPlatform,
+      // home: FutureBuilder(
+      //   future: Firebase.initializeApp(
+      //     options: DefaultFirebaseOptions.currentPlatform,
```

Comment:
Can't this just be removed or added properly?

Reply:
It's the example, I thought so first as well, but it is to have a quick firebase example. Ideally, we would add a proper firebase example, but for now I do not want to remove it. It's a simple formatting here